fapi-client/src/main/scala/com/gu/facia/api/utils/ExternalLinks.scala

```diff
@@ -0,0 +1,32 @@
+package com.gu.facia.api.utils
+
+import java.net.URI
+
+import scala.util.Try
+
+object ExternalLinks {
+  val corsOrigins = List(
+    "http://www.theguardian.com",
+    "https://www.theguardian.com",
+    "https://profile.theguardian.com",
+    "https://composer.gutools.co.uk",
+    "https://composer.release.dev-gutools.co.uk",
+    "https://composer.code.dev-gutools.co.uk",
+    "http://preview.gutools.co.uk",
+    "https://preview.gutools.co.uk")
+
+  val GuardianDomains = corsOrigins flatMap { uri =>
```

Comment:
Why's this a capital `G`? This may be complex enough to benefit from a type annotation.